data.sql

```diff
@@ -42,3 +42,42 @@ SET owner_id = CASE
     WHEN name = 'Charmander' OR name = 'Squirtle' OR name = 'Blossom' THEN (SELECT id FROM owners WHERE full_name = 'Melody Pond')
     WHEN name = 'Angemon' OR name = 'Boarmon' THEN (SELECT id FROM owners WHERE full_name = 'Dean Winchester')
 END;
+
+INSERT INTO vets (name, age, date_of_graduation)
+ VALUES ('William Tatcher', 45, '2000-04-23'),
+ ('Maisy Smith', 26, '2019-01-17'),
+ ('Stephanie Mendez', 64, '1981-05-04'),
+ ('Jack Harkness', 38, '2008-06-08');
+
+SELECT id FROM vets WHERE name = 'William Tatcher';
+SELECT id FROM species WHERE name = 'Pokemon';
```

Comment:
### Optional
- [ ] It would be nice if the `data.sql` file saves everything that populates table with data only, so it would be nice if we move the `SELECT` statement into the `queries.sql` file instead

Reply:
Select statement removed from data.sql into queries.sql👍